lua/diagnostics.lua

```diff
@@ -0,0 +1,46 @@
+local module = {}
```

Comment:
In this particular case I'm okay with having a little bit of Lua code, as we might as well. Typically I have a strong preference for nearly 100% Vim Script.